tuna/miopen/miopen_lib.py

```diff
@@ -334,7 +333,7 @@ def compose_worker_list(self, res, args):
           num_procs = int(env['slurm_cpus'])
         else:
           # JD: This sould be the responsibility of the machine class
-          num_procs = int(machine.get_num_cpus())
+          num_procs = int(machine.get_num_cpus() * .6)
```

Comment:
What is the reason for scaling down here ?

Reply:
I was encountering some deadlocking during testing. My theory was too many compilers. Scaling back the workers was helping.